### PR TITLE
MAINT: fix linter on master

### PR DESCRIPTION
**Pull request by rlucas7**
#### Reference issue
N/A

#### What does this implement/fix?
When I merged 
https://github.com/scipy/scipy/pull/12582
I missed a whitespace that causes linter to fail. 
This PR removes the whitespace character that offends the linter.

### CODE REVIEW

**Review by tylerjereddy — APPROVED** [review]
Fixes the `master` branch failure for `pycodestyle scipy benchmarks/benchmarks` when I check this branch locally.

Feel free to self-merge; I guess you can wait for CI to flush through if you want.

**Comment by rlucas7** [comment]
Thanks @tylerjereddy
I double checked and the multivariate t PR was all green when merged. 

Is the linter only running on master branch-after merges? 

-L

**Pull request merged**
